boards/nordic/nrf54h20dk/Kconfig.defconfig

```diff
@@ -27,3 +27,10 @@ config FLASH_LOAD_OFFSET
 	default 0x2c000 if !USE_DT_CODE_PARTITION
 
 endif # BOARD_NRF54H20DK_NRF54H20_CPUAPP_IRON
+
+if BOARD_NRF54H20DK_NRF54H20_CPUAPP_IRON_MCUBOOT_B0
```

Comment:
I do not have strong opinion, but which of those two you'd consider better and why?
``BOARD_NRF54H20DK_NRF54H20_CPUAPP_IRON_MCUBOOT_B0`` vs ``BOARD_NRF54H20DK_NRF54H20_CPUAPP_IRON_B0_MCUBOOT``

Reply:
I prefer `MCUBOOT_B0`, because I assume mcuboot is the more popular choice, thus it seems better to treat B0 as an addition to MCUBOOT. Also, it seems the main usage of B0 is allowing MCUBOOT upgrade